deployment/docker-compose.yml

```diff
@@ -92,6 +92,7 @@ services:
       RADIX_NG_API__DisableCoreApiHttpsCertificateChecks: "${DISABLE_CORE_API_CERTIFICATE_CHECKS}"
       RADIX_NG_API__NetworkName: "${NETWORK_NAME}"
       RADIX_NG_API__EnableSwagger: "${ENABLE_SWAGGER}"
+      RADIX_NG_API__MaxPageSize: "${MAX_PAGE_SIZE}"
```

Comment:
I tend to add it to the appSettings.json too so people can see all the settings there :)